zgw/zaken-api/src/test/kotlin/com/ritense/zakenapi/link/ZaakInstanceLinkServiceTest.kt

```diff
@@ -0,0 +1,28 @@
+package com.ritense.zakenapi.link
+
+import com.ritense.zakenapi.repository.ZaakInstanceLinkRepository
+import org.junit.jupiter.api.Assertions.*
```

Comment:
Please write out all import instead of using `*`

Also, missing copyright header